### ISSUE TITLE
only full ways get highlighted, even if only parts of them are covered

### ISSUE THREAD

**Issue by joshinils** [kartaview/openstreetcam.org#327]
For example, I made a U-Turn on my bikle on https://www.openstreetmap.org/way/184049387 before it bends.
On https://kartaview.org/map/@52.6544863276543,13.28101225889236,18z it gets painted in blue all the way.

**Comment by alex-grab** [comment]
yep, can confirm this is the snapping-to-road behaviour – partial way coverage is not supported yet but on the roadmap

**Comment by joshinils** [comment]
For reference #327:
> In comparison to [wandrer.earth](https://wandrer.earth), i do not know which ways will get painted in blue on the map after i upload a sequence taken there.
> 
> On wandrer.earth I can enable untravelled ways on my [Big Map](https://wandrer.earth/dashboard/map) (login required), I would like this for KartaView aswell.
